client/src/pages/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,75 @@
+
+.LoginPageMain {
```

Comment:
You've got a good start on these class names, but they're not exactly following BEM. I think you want something like `.Login` here. Other examples would be like `.Login__form-container` or `.Login__form`.